src/query/plan/preprocess.hpp

```diff
@@ -287,6 +286,8 @@ struct FilterInfo {
 /// expressions that should be generated.
 class Filters final {
  public:
+  std::vector<FilterInfo> all_filters_;
```

Comment:
This is not such a good practice... It is better than to create a getter method than to put field as a private

Reply:
I needed it because of [this line](https://github.com/memgraph/memgraph/blob/dfef16b8bfd621ef363cd81d4ffe584a7c727732/tests/unit/query_plan_operator_to_string.cpp#L235-L236). Access to `all_filters_` is already possible via the public iterators, will add a setter for it.